doc/notebooks/jupyter_tutorial.ipynb

```diff
@@ -451,12 +486,27 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "series = gj.SeriesMap(height=500)\n",
-    "series.add_rasters([\"elevation\", \"elevation_shade\", \"slope\"])\n",
-    "# series.add_vectors([\"streams\", \"streets\", \"viewpoints\"])\n",
-    "series.d_vect(map=\"streets\")\n",
-    "series.d_barscale()\n",
-    "series.show()  # Create Slider"
+    "cores_range = range(1, 13)\n",
+    "\n",
+    "benchmark_results = {}\n",
+    "\n",
+    "for cores in cores_range:\n",
+    "    gs.run_command(\"g.gisenv\", set=f\"NPROCS={cores}\")\n",
+    "\n",
+    "    start_time = time.time()\n",
+    "\n",
+    "    series = gj.SeriesMap(height=500)\n",
+    "    series.add_rasters([\"elevation\", \"elevation_shade\", \"slope\"])\n",
+    "    series.add_vectors([\"streams\", \"streets\", \"viewpoints\"])\n",
+    "    series.d_vect(map=\"streets\")\n",
+    "    series.d_barscale()\n",
+    "    series.show()  # Create Slider\n",
+    "\n",
+    "    end_time = time.time()\n",
+    "    elapsed_time = end_time - start_time\n",
+    "\n",
+    "    benchmark_results[cores] = elapsed_time\n",
+    "    print(f\"Cores: {cores}, Time: {elapsed_time:.2f} seconds\")"
```

Comment:
I think I wouldn't present it as a benchmark, since this is supposed to be documentation for a user. I would put this in the report. Here, I would just inform user SeriesMap is using multiple cores for rendering and if they want to set a specific number, they can use NPROCS variable, and you can show an examlpe how to set that.

Reply:
Done.